controller/DeliveryMgmt.php

```diff
@@ -54,6 +55,9 @@ class DeliveryMgmt extends \tao_actions_SaSModule
 {
     use TaskLogActionTrait;
 
+    private const FEATURE_FLAG_GROUPS_DISABLED = 'FEATURE_FLAG_GROUPS_DISABLED';
```

Comment:
```suggestion
    private const FEATURE_FLAG_GROUPS_LINK_DISABLED = 'FEATURE_FLAG_GROUPS_DISABLED';
```

Reply:
Why link?

Reply:
I just wanted to expose the fact that this ff is hiding only the assignee part. Maybe something like `FEATURE_FLAG_GROUPS_ASSIGNER_DISABLED`... correct if I am wrong.